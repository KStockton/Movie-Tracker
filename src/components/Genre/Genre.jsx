import React from 'react'

const Genre = () => {
  return (
    <section className="genre-container">
      <h1>genre</h1>
    </section>
  )
}
export default Genre