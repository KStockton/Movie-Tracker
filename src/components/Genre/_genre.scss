.genre-container {
    background-color: rgb(36, 75, 100);
    height: 100%;
    color: white;
}