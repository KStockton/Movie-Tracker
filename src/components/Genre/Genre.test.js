import React from 'react';
import Genre from './Genre';
import { shallow } from 'enzyme'

describe('Genre', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow (<Genre/>)
  })

  it('Genre component should match the snapshot', () =>{
    expect(wrapper).toMatchSnapshot()
  })


})