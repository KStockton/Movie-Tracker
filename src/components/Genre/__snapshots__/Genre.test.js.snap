// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Genre Genre component should match the snapshot 1`] = `ShallowWrapper {}`;
